lib/howitzer/web/page_dsl.rb

```diff
@@ -0,0 +1,42 @@
+module Howitzer
+  module Web
+    # This module combines page dsl methods
+    module PageDsl
+      # This class is for private usage only
+      class PageScope
+        def initialize(page_klass, &block)
+          self.page_klass = page_klass
+          instance_eval(&block) if block_given?
```

Comment:
because #open is not using this logic anymore, I suggest to remove "if block_given?" in order to make the block required